CHANGELOG.md

```diff
@@ -89,5 +89,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix unhandled Promise rejection in DefaultScreenSharingSession#start
 - Fix canary deployment script
 - Fix sip call integration test
+- Add stage to saucelabs session name for integration tests
```

Comment:
This section is for 1.1.0. Please add it to the "Unreleased" section (line 17).

Reply:
oh yes will fix it